app/helpers/news_helper.rb

```diff
@@ -16,11 +7,22 @@ def preload_news_articles(query_root = NewsArticle)
     query_root
       .preload([
         {:section  => {:home_node => :parent}},
-        {:sections => {:home_node => :parent}}
-      ])
+        {:sections => {:home_node => :parent}}])
       .published
       .by_release_date
       .by_published_at
       .by_name
   end
+
+
+  # List of Ministers for filtering
+  def ministers_list
+    Minister.order(:name).decorate.sort_by! { |minister| minister.sort_order }
```

Comment:
This is a good point, however we should bundle it into a performance tech-debt card.